### DISCUSSION TITLE
第2节课作业(5班)

### DISCUSSION THREAD

**Discussion by sanbuphy** [InternLM/Tutorial#124] [作业]
宝子们在这里提交你的作业

**Comment by Milkve** [comment]
大模型第二节课作业：[[学习笔记] 轻松分钟玩转书生·浦语大模型趣味 Demo](https://juejin.cn/spost/7320456973899644928)
[300字小故事](https://juejin.cn/post/7320456973899644928#heading-8)
[下载 InternLM-20B 的 config.json 文件到本地](https://juejin.cn/post/7320456973899644928#heading-21)
[浦语·灵笔的图文理解及创作部署](https://juejin.cn/post/7320456973899644928#heading-13)
[Lagent 工具调用 Demo 创作部署](https://juejin.cn/post/7320456973899644928#heading-12)

**Comment by shengmouget** [comment]
[第二课作业](https://github.com/shengmouget/LLM_Job?tab=readme-ov-file#%E7%AC%AC%E4%B8%80%E6%AC%A1%E4%BD%9C%E4%B8%9A)

**Comment by JayceNing** [comment]
**基础作业**
* 使用 InternLM-Chat-7B 模型生成 300 字的小故事
![image](https://github.com/InternLM/tutorial/assets/44694099/aa749932-cc4b-475e-bf16-2991c283ff6b)

* 熟悉 hugging face 下载功能，使用 huggingface_hub python 包，下载 InternLM-20B 的 config.json 文件到本地

网不好，下载不下来。网要是好的话这行代码没问题。

![image](https://github.com/InternLM/tutorial/assets/44694099/e4991ebf-d339-4e27-ba77-409f71abe483)


**进阶作业**
* 完成浦语·灵笔的图文理解及创作部署
![image](https://github.com/InternLM/tutorial/assets/44694099/9705aa38-1e4f-4733-be7f-0a451c06d5a9)

* 完成 Lagent 工具调用 Demo 创作部署
![image](https://github.com/InternLM/tutorial/assets/44694099/3ceb1cae-a394-4a31-9be2-3adcf8c693c4)

**Reply by JayceNing** [reply]
huggingface库由于网络原因下载不下来config文件，但是可以用OpenXLab库下载
![image](https://github.com/InternLM/tutorial/assets/44694099/1a81c0a7-4595-41dd-8da4-478cac519ca1)

**Comment by ZhaoHongSen0726** [comment]
作业和笔记放一起了：https://blog.csdn.net/m0_75085274/article/details/135425532?spm=1001.2014.3001.5502

**Reply by ZhaoHongSen0726** [reply]
config文件因为网络原因，选择使用 ModelScope进行下载

**Comment by DreamBinary** [comment]
[https://github.com/DreamBinary/InternLMCamp/tree/master/day02/work](https://github.com/DreamBinary/InternLMCamp/tree/master/day02/work)

**Comment by GitEasonXu** [comment]
第2节：轻松玩转书生·浦语大模型趣味 Demo作业
https://github.com/GitEasonXu/InternLM/blob/main/homework/course_2/coursenote.md

**Comment by Farewell-CK** [comment]
作业：https://github.com/8baby8/Farewell_Learn_puyu/blob/main/%E7%AC%AC%E4%BA%8C%E8%8A%82%E8%AF%BE/%E7%AC%AC%E4%BA%8C%E8%8A%82%E8%AF%BE%E4%BD%9C%E4%B8%9A.ipynb

**Comment by lgkkey** [comment]
作业：https://github.com/lgkkey/LM_Study-Record/blob/main/work/1/README.md

**Comment by Beforcer** [comment]
# 下载config
![作业下载config](https://github.com/InternLM/tutorial/assets/86581980/8643fead-6c83-4c38-a8c5-0fd6f38b7bf7)
# 小故事
![作业小故事](https://github.com/InternLM/tutorial/assets/86581980/08cef3bc-18c2-4810-813d-235be42cd4f8)

**Comment by Egber1t** [comment]
书生浦语Day2
https://blog.csdn.net/qq_40167280/article/details/135479104?spm=1001.2014.3001.5501

**Comment by Lengxiaom0** [comment]
https://blog.csdn.net/lxmlxmmmm/article/details/135488896?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135488896%22%2C%22source%22%3A%22lxmlxmmmm%22%7D

**Comment by ATang0729** [comment]
作业：https://github.com/ATang0729/InternLMCamp_Notes/blob/main/notes/class2-HW/HW-class2.md

**Comment by preordinary** [comment]
https://blog.csdn.net/preor/article/details/135468849?spm=1001.2014.3001.5502

**Comment by sanbuphy** [comment]
https://zhuanlan.zhihu.com/p/680168238?
